README.md

```diff
@@ -1 +1,220 @@
-"react-todo-app" is a tool that helps to organize a day.
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<!-- TABLE OF CONTENTS -->
+
+# <b>Table of Contents 📗
+
+- [About the Project 📖](#about-project)
+  - [Built With](#built-with) 🛠
+    - [Tech Stack 🗜️](#tech-stack)
+    - [Key Features 🏷️](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [Getting Started 💻](#getting-started)
+  - [Setup ⚙️](#setup)
+  - [Prerequisites 🧱](#prerequisites)
+  - [Install 🏗️](#install)
+  - [Usage 📂](#usage)
+  - [Run tests 🧪](#run-tests)
+  - [Deployment](#deployment)
+- [Author 👤](#author)
+- [Future Features 🔭](#future-features)
+- [Contributing 🤝](#contributing)
+- [Show your support 🌟](#support)
+- [Acknowledgements 🙏](#acknowledgements)
+- [License 📝](#license)
+</b>
+<hr><hr>
+<!-- PROJECT DESCRIPTION -->
+
+# <b>React Todos  📖</b><a name="about-project"></a>
+
+**React Todos** is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+<br><hr>
+
+<!-- BUILT WITH -->
+## <b>Built With 🛠</b><a name="built-with"></a><br>
+
+<!-- TECH STACK -->
+### <b>Tech Stack 🗜️</b><a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML5</a> markup to build raw structure of this web page</li><br>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS3</a> custom properties<br>
+    <li><a href="https://reactjs.org/">React Js</a></li><br>
+    <li><a href="https://mozilla.github.io/addons-linter/">Linters</a> for coding convention and coding formating</li>
+  </ul>
+</details>
+
+<br><hr>
+
+<!-- KEY FEATURES -->
+### <b>Key Features 🏷️</b> <a name="key-features"></a>
+
+- **Adding a new item**
+- **Removing a selected item**
+- **Marking a selected item as complete**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<br>
+<hr>
+
+<!-- LIVE DEMO -->
+## <b>Live Demo 🚀</b><a name="live-demo"></a>
+
+> This app will be deployed in the near future.
+
```

Comment:
- Kindly try to add the link of live demo to your project README.md file or remove the live demo section from README.